make/jdk/src/classes/build/tools/module/GenModuleInfoSource.java

```diff
@@ -473,6 +473,11 @@ private void process(Parser parser, boolean extraFile) throws IOException {
                     if (parser.peekToken() != null) {  // must be EOF
                         throw parser.newError("is malformed");
                     }
+                } else if (token.equals("import")) {
```

Comment:
why is this change necessary?

Reply:
It allows to use imports in module infos, so annotations do not need to use FQN.